### PR TITLE
S02 us14 login

### PR DESCRIPTION
**Pull request by bouboumael**

### CODE REVIEW

**Review by RedPandore — DISMISSED** [review]

File: src/Controller/SecurityController.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class SecurityController extends AbstractController
+{
+    /**
+     * @Route("/login", name="app_login")
+     */
+    public function login(AuthenticationUtils $authenticationUtils): Response
+    {
+        // if ($this->getUser()) {
+        //     return $this->redirectToRoute('target_path');
+        // }
```
Comment (LucyMerlier) [inline src/Controller/SecurityController.php]: commented code

File: src/Security/AppAuthenticator.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                new CsrfTokenBadge('authenticate', $request->get('_csrf_token')),
+            ]
+        );
+    }
+
+    public function onAuthenticationSuccess(Request $request, TokenInterface $token, string $firewallName): ?Response
+    {
+        /*if ($targetPath = $this->getTargetPath($request->getSession(), $firewallName)) {
+            return new RedirectResponse($targetPath);
+        }*/
```
Comment (LucyMerlier) [inline src/Security/AppAuthenticator.php]: commented code

File: templates/includes/_login.html.twig [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<a class="nav-link" data-bs-toggle="modal" data-bs-target="#loginModal" href="#">
+    <i class="fas fa-user"></i>
+    Connexion</a>
```
Comment (LucyMerlier) [inline templates/includes/_login.html.twig]: a tag should be on line below

File: templates/includes/_login.html.twig [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+<!-- Modal -->
+<div class="modal fade" id="loginModal" tabindex="-1" aria-labelledby="loginModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLabel">Se connecter</h5>
```
Comment (LucyMerlier) [inline templates/includes/_login.html.twig:11]: honest question : is this h5 semantic? are there h2, h3 and h4 before?

File: templates/security/login.html.twig [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    {#
+        Uncomment this section and add a remember_me option below your firewall to activate remember me functionality.
+        See https://symfony.com/doc/current/security/remember_me.html
+
+        <div class="checkbox mb-3">
+            <label>
+                <input type="checkbox" name="_remember_me"> Remember me
+            </label>
+        </div>
+    #}
```
Comment (LucyMerlier) [inline templates/security/login.html.twig]: commented code

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

File: templates/includes/_login.html.twig (hunk 3 above)
Comment (bouboumael) [reply templates/includes/_login.html.twig]: done

File: templates/includes/_login.html.twig (hunk 4 above)
Comment (RaphaelBS-WCS) [reply templates/includes/_login.html.twig:11]: Yes

File: src/Security/AppAuthenticator.php (hunk 2 above)
Comment (bouboumael) [reply src/Security/AppAuthenticator.php]: done

**Review by wonecode — APPROVED** [review]

**Review by RedPandore — APPROVED** [review]

**Review by RaphaelBS-WCS — APPROVED** [review]
Review

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**